### PR TITLE
Update wordings for collaborator invitation email

### PR DESCRIPTION
**Pull request by IniZio**
ref: #2672

**Preview**

> The project name is "accounts"

**Subject**

`"You are invited to collaborate on "accounts" in Authgear"`

**Text body**

```
Join \"accounts\" as a collaborator\n\nYou have been invited to join as a collaborator to manage the project \"accounts\" in Authgear. Please click the link below to view the invitation.\n\nhttp://portal.localhost:8000/collaborators/invitation?code=RMW389CRP5N7WHKPJF5860EB166JJ71H\n\nAuthgear is a user identity and authentication software that helps you manage users and log them in securely.\n\nIf you want to reject the invitation please ignore this email.\n
```

**HTML preview**

Screen Shot 2023-01-03 at 15 44 17

### CODE REVIEW

File: resources/portal/templates/en/messages/collaborator_invitation_email.mjml [hunk 1]
```diff
@@ -9,9 +9,12 @@
   <mj-section>
     <mj-column>
       <mj-text font-weight="bold" font-size="24px" padding="20px">Join {{ .AppName }} as a collaborator</mj-text>
-      <mj-text font-size="16px" padding="20px">You have been invited to join {{ .AppName }} as a collaborator. Please click the link below to view the invitation.</mj-text>
+      <mj-text font-size="16px" padding="20px">You have been invited to join {{ .AppName }} as a collaborator to manage the [project] account in Authgear. Please click the link below to view the invitation.</mj-text>
```
Comment (louischan-oursky) [inline resources/portal/templates/en/messages/collaborator_invitation_email.mjml:12]: `[project]` actually means substituting the app ID here. 😂 

`{{ .AppName }}` does not appear after `join` too.

Another thing to put attention is that we want to see the app ID here. Please double check if `{{ .AppName }}` refers to the app ID or the app name. We have both. The app name is defined in the translation.json under the key `app.name`.

**Comment by louischan-oursky** [comment]
You only need to update the .mjml files. The .html is generated by running `make html-email`.

File: resources/portal/templates/en/messages/collaborator_invitation_email.mjml (hunk 1 above)
Comment (IniZio) [reply resources/portal/templates/en/messages/collaborator_invitation_email.mjml]: Currently the render uses project id, pasted an example for reference. Also fixed `[project]` 🤦

**Comment by IniZio** [comment]
Yes I did generate the html with `make html-email`

**Comment by louischan-oursky** [comment]
@fungc-io Do you think we need to apply some style to the app ID to make it more visually prominent in the subject and in the body?

**Comment by fungc-io** [comment]
yea, bold it will do

**Comment by louischan-oursky** [comment]
I think we can make the app ID bold in the body, but not in the subject. Maybe we can quote it in the subject. Which delimiter should we use?

**Comment by fungc-io** [comment]
double quote `" U+0022`

**Comment by IniZio** [comment]
Updated to bold the project ID in html body, added quotes for other places where bold style cannot be applied.

**Pull request merged**